Would be nice to have this in for the tutorial, but need to make sure opam-repo updates before then:
```
$ opam update
$ opam sw create 5.1.0~rc3+tsan
[ERROR] No compiler matching `5.1.0~rc3+tsan' found, use `opam switch list-available' to see what is available, or use `--packages' to select packages
        explicitly.
```

--- a/doc/prereqs.md
+++ b/doc/prereqs.md
@@ -3,13 +3,13 @@
 You will need OCaml 5.1 or later, which can be installed using [opam](https://opam.ocaml.org/):
 
 ```sh
-opam switch create 5.1.0~rc2
+opam switch create 5.1.0~rc3
 ```
 
 For profiling with `perf` (Linux-only), it may be helpful to use a compiler with frame-pointers enabled instead, like this:
 
 ```sh
-opam switch create 5.1.0~rc2+fp ocaml-variants.5.1.0~rc2+options ocaml-option-fp
+opam switch create 5.1.0~rc3+fp ocaml-variants.5.1.0~rc3+options ocaml-option-fp
 ```
 
 This repository uses Git submodules. Make sure they're enabled with:

